README-Networking.md

```diff
@@ -33,9 +39,32 @@ CNI-Genie is a CNI plugin that enables Kubernetes to simultaneously have access
   - Service Discovery
     - micro DNS at each node, DNS lookups always local
   - Multicast Networking - UDP multcast even if underlying network doesn't support it
-  - 
+
 **Flannel**
 
+Try it out: `kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml`
+  - Focused on networking. For network policy, other projects such as Calico can be used
+  -  Main advantage of this model is that it removes the port mapping complexities that come from sharing a single host IP
+  - Does not control how containers are networked to the host, only how the traffic is transported between hosts
+  - Service Discovery
+    - Runs a small, single binary agent called `flanneld` on each host, and is responsible for allocating a subnet lease to each host out of a larger, preconfigured address space
+    - uses either the Kubernetes API or etcd directly to store the network configuration, the allocated subnets, and any auxiliary data (such as the host's public IP)
+
 **Calico**
 
-**Romana**
\ No newline at end of file
+Try it out: `kubectl apply -f \
+https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/rbac.yaml`
+
+  - Enables networking and network policy
```

Comment:
Al of the CNIs enable networking right, so you can remove that?

Reply:
not all enable/have network policy options.  flannel doesn't do the policy part for example.

Reply:
right but all have networking so how about just tweak to Enables network policy?

Reply:
Maybe not.

"Flannel does not control how containers are networked to the host, only how the traffic is transported between hosts. However, flannel does provide a CNI plugin for Kubernetes and a guidance on integrating with Docker."

So I am confused on if you need a separate plug-in for networking.

Reply:
https://github.com/coreos/flannel-cni

Reply:
Nevermind, since all of the networking plugins here don't necessarily provide networking (though I think that means they are not a CNI)